docs/get-started/vue-vite.md

```diff
@@ -0,0 +1,268 @@
+
+It also supports both default and named component exports.
+
+### Prop description and JSDoc tag annotations
+
+To provide a description for a prop, including tags, you can use JSDoc comments in your component's props definition:
+
+```html
+<!-- YourComponent.vue -->
+<script setup lang="ts">
+  interface MyComponentProps {
+    /**
+     * The name of the user
+     */
+    name: string;
+    /**
+     * The category of the component
+     * @required
+     * @default 'Uncategorized'
+     */
+    category: string;
+  }
+</script>
+```
+
+### Events types extraction
+
+<Callout variant="info">
+
+Due to technical limitations of Volar / vue language features, JSDoc descriptions for emitted events can not be extracted.
+
+</Callout>
+
+To provide a type for an emitted event, you can use TypeScript types in your component's `defineEmits` call:
+
+```html
+<!-- YourComponent.vue -->
+<script setup lang="ts">
+  type MyFooEvent = 'foo';
+
+  interface MyEvents {
+    (event: MyFooEvent, data?: { foo: string }): void;
+    (event: 'bar', value: { year: number; title?: any }): void;
+    (e: 'baz'): void;
+  }
+
+  const emit = defineEmits<MyEvents>();
+</script>
+```
+
+Which will generate the following controls:
+
+![Controls generated from events](./vue-component-meta-event-types-controls.png)
+
+### Slots types extraction
+
+The slot types are automatically extracted from your component definition and displayed in the controls panel.
+
+```html
+<!-- YourComponent.vue -->
+<template>
+  <slot name="no-bind"></slot>
+  <br />
+  <slot :num="123"></slot>
+  <br />
+  <slot name="named" str="str"></slot>
+  <br />
+  <slot name="vbind" v-bind="{ num: 123, str: 'str' }"></slot>
+</template>
+
+<script setup lang="ts"></script>
+```
+
+The definition above will generate the following controls:
+
+![Controls generated from slots](./vue-component-meta-slot-types-controls.png)
+
+### Exposed properties and methods
+
+The properties and methods exposed by your component are automatically extracted and displayed in the controls panel.
+
+```html
+<!-- YourComponent.vue -->
+<script setup lang="ts">
+  import { ref } from 'vue';
+
+  const label = ref('Button');
+  const count = ref(100);
+
+  defineExpose({
+    /**
+     * a label string
+     */
+    label,
+    /**
+     * a count number
+     */
+    count,
+  });
+</script>
+```
+
+The definition above will generate the following controls:
+
+![Controls generated from exposed properties and methods](./vue-component-meta-exposed-types-controls.png)
+
+## API
+
+### Options
+
+You can pass an options object for additional configuration if needed:
+
+```ts
+// .storybook/main.ts
+import type { StorybookConfig } from '@storybook/vue3-vite';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/vue3-vite',
+    options: {
+      docgen: 'vue-component-meta',
+    },
+  },
+};
+
+export default config;
+```
+
+#### `docgen`
```

Comment:
This is only supported since version 8, right? At the top of the file it says `>= 7`

Reply:
Good call. The framework itself is supported from 7+, but this API is only in 8+. I've clarified that.